cmd/collectors/zapi/collector/zapi.go

```diff
@@ -412,6 +414,111 @@ func (me *Zapi) PollData() (*matrix.Matrix, error) {
 	return me.Matrix, nil
 }
 
+func (me *Zapi) CollectAutoSupport(p *collector.Payload) {
+	var exporterTypes []string
+	for _, exporter := range me.Exporters {
+		exporterTypes = append(exporterTypes, exporter.GetClass())
+	}
+
+	var counters = make([]string, 0)
+	if me.Params != nil {
+		c := me.Params.GetChildS("counters")
+		c.FlatList(&counters, "")
+	}
+
+	var schedules = make([]collector.Schedule, 0)
+	tasks := me.Params.GetChildS("schedule")
+	if tasks != nil && len(tasks.GetChildren()) > 0 {
+		for _, task := range tasks.GetChildren() {
+			schedules = append(schedules, collector.Schedule{
+				Name:     task.GetNameS(),
+				Schedule: task.GetContentS(),
+			})
+		}
+	}
+
+	clientTimeout := ""
+	clientNode := me.Params.GetChildS("client_timeout")
```

Comment:
if client_timeout is not specified in config, it gives ""(tested this change with my local). can we use DefaultTimeout?
https://github.com/NetApp/harvest/blob/main/pkg/api/ontapi/zapi/client.go#L135